core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/controller/Controller.scala

```diff
@@ -866,6 +881,18 @@ class Controller(
           3
         )
       } else {
+        eventListener.workflowExecutionErrorListener.apply(
+          ErrorOccurred(
+            Error(
+              "Breakpoint target operator not found",
+              "Engine:Controller:PassBreakpointTo",
+              Map(
+                "trace" -> Thread.currentThread().getStackTrace().mkString("\n"),
+                "faulty_op" -> opTag.getGlobalIdentity
```

Comment:
In `Error.convertToMap()`, we use camel case for the key of the map but here we use underscore. I suggest we always use camel case for the key.